src/compiler/emitter.ts

```diff
@@ -4716,6 +4719,9 @@ if (typeof __param !== "function") __param = function (paramIndex, decorator) {
                     write(unaliasedModuleNames.join(", "));
                 }
                 write("], function (require, exports");
+                if (compilerOptions.noEmitHelpers) {

```

Comment:
We are considering a different approach to making `__extends` and other helpers available with something like `--noEmitHelpers`, so I would recommend removing this line.